### PR TITLE
feat: add support to auth module 0.3.0

### PR DESCRIPTION
**Pull request by Al-Pragliola**

### CODE REVIEW

**Review by ralgozino — CHANGES_REQUESTED** [review]
Dex branding changes LGTM, but there are some other changes for gangplank included in this PR, is that ok?
Also, this PR is for changes that are still in draft or am I wrong?
- https://github.com/sighupio/fury-kubernetes-auth/pull/27

**Comment by Al-Pragliola** [comment]
> Dex branding changes LGTM, but there are some other changes for gangplank included in this PR, is that ok? Also, this PR is for changes top the auth module that are still in draft or am I wrong?
> 
> * [feat: prepare v0.3.0 fury-kubernetes-auth#27](https://github.com/sighupio/fury-kubernetes-auth/pull/27)

change the PR's title to "feat: add support to auth module 0.3.0", same goes for the PR you linked that is not in draft state anymore, thanks for pointing it out

File: schemas/public/onpremises-kfd-v1alpha2.json [hunk 1]
```diff
@@ -1602,6 +1602,12 @@
         },
         "sessionSecurityKey": {
           "type": "string"
+        },
+        "removeCAFromKubeconfig": {
+          "type": "boolean"
+        },
+        "namespace": {
+          "type": "string"
```
Comment (ralgozino) [inline schemas/public/onpremises-kfd-v1alpha2.json:1610]: let's add a description to these fields and we are good to go :)

**Review by ralgozino — CHANGES_REQUESTED** [review]

**Review by ralgozino — APPROVED** [review]
lgtm

**Pull request merged**